src/styles/main.scss

```diff
@@ -201,6 +220,16 @@
 .key-value-item {
   box-sizing: border-box;
   line-height: 1.5rem;
+
+  p {
+    margin-block-start: 0px;
+    margin-block-end: 0px;
+  }
+}
+
+
+div[data-testid="stModal"] > div > div {
+  background-color: red;  // If the modals should have a different background color, set it here
```

Comment:
Note: Here you can set dialog background color

Reply:
@shu echoing what Kim said. I just copied in and set to white :)